### DISCUSSION TITLE
no write access to repository

### DISCUSSION THREAD

**Discussion by pauladesimone** [community/community#107538] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![Screenshot 2024-02-10 220017](https://github.com/community/community/assets/145982787/30325959-ded1-4764-9b39-709064dbc22d)

been getting this message after cloning a repository in which I am a collaborator. Why is that happening?

**Comment by naveenalla3000** [comment]
ok, first you fork the repo and the make updates and push to your github.

**Reply by pauladesimone** [reply]
Hi! Thanks for your answer. We actually need to be all wprking on the same repository, so I would need to clone it. I was working with a fork before but had to change it cause it wasn't helping.
